### PR TITLE
DEVEL_ONLY: Allow accepting unsigned pkgs

### PR DESCRIPTION
**Pull request by Rezney**

### CODE REVIEW

**Comment by pirat89** [comment]
@Rezney : I think the name "accept_unsigned" could be misleading a little. Someone would point out that they want to do something with unsigned RPMs (install durign the transaction for example) and they could think they need to use the envar. Instead of that, what about `LEAPP_DEVEL_RPMS_ALL_SIGNED`

**Comment by Rezney** [comment]
@pirat89  "LEAPP_DEVEL_RPMS_ALL_SIGNED" sounds fair

File: repos/system_upgrade/el7toel8/actors/redhatsignedrpmscanner/tests/test_redhatsignedrpmscanner.py [hunk 1]
```diff
@@ -104,7 +124,7 @@ def test_has_package(current_actor_context):
     ]
 
     current_actor_context.feed(InstalledRPM(items=installed_rpm))
-    current_actor_context.run()
+    current_actor_context.run(config=mock_configs.CONFIG)
```
Comment (jmikovic) [inline repos/system_upgrade/el7toel8/actors/redhatsignedrpmscanner/tests/test_redhatsignedrpmscanner.py]:
```suggestion
    current_actor_context.run(config_model=mock_configs.CONFIG)
```

**Comment by jmikovic** [comment]
leapp-ci build

File: repos/system_upgrade/el7toel8/actors/redhatsignedrpmscanner/tests/test_redhatsignedrpmscanner.py (hunk 1 above)
Comment (Rezney) [reply repos/system_upgrade/el7toel8/actors/redhatsignedrpmscanner/tests/test_redhatsignedrpmscanner.py]: Fixed, thanks.

File: repos/system_upgrade/el7toel8/libraries/config/mock_configs.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+from leapp.models import IPUConfig, EnvVar, OSRelease, Version
+
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/libraries/config/mock_configs.py]: Maybe little too late, but couldn't be better to put a discaimer into the file? I knot that there is *mock_* prefix in the name, but to ensure that people will not use it in the standard code

Comment (bocekm) [reply repos/system_upgrade/el7toel8/libraries/config/mock_configs.py]: I'm about to merge #340 in a while.

**Pull request merged**